ribbon/src/main/java/feign/ribbon/RibbonClient.java

```diff
@@ -4,36 +4,53 @@
 import java.net.URI;
 
 import com.netflix.client.ClientException;
-import com.netflix.client.ClientFactory;
 import com.netflix.client.config.CommonClientConfigKey;
 import com.netflix.client.config.DefaultClientConfigImpl;
-import com.netflix.client.config.IClientConfig;
-import com.netflix.loadbalancer.ILoadBalancer;
 
 import feign.Client;
 import feign.Request;
 import feign.Response;
 
 /**
- * RibbonClient can be used in Fiegn builder to activate smart routing and resiliency capabilities
+ * RibbonClient can be used in Feign builder to activate smart routing and resiliency capabilities
  * provided by Ribbon. Ex.
+ * 
  * <pre>
- * MyService api = Feign.builder.client(new RibbonClient()).target(MyService.class,
- * "http://myAppProd");
+ * MyService api = Feign.builder.client(RibbonClient.builder().build())
+ *     .target(MyService.class, &quot;http://myAppProd&quot;);
  * </pre>
+ * 
  * Where {@code myAppProd} is the ribbon client name and {@code myAppProd.ribbon.listOfServers}
  * configuration is set.
  */
 public class RibbonClient implements Client {
 
   private final Client delegate;
+  private final LBClientFactory lbClientFactory;
 
+  public static Builder builder() {
+    return new Builder();
+  }
+
+  /**
+   * @deprecated Use the {@link RibbonClient#builder()}
```

Comment:
`RibbonClient.builder().build()` is onerous for the simple case. how about just `RibbonClient.create()`?

```java
 public static RibbonClient create() {
   return new RibbonClient.builder().build();
 }
```